### DISCUSSION TITLE
flashing digital display - heat meter pollucom f

### DISCUSSION THREAD

**Discussion by stefanovich1** [jomjol/AI-on-the-edge-device#3407] [Q&A]
I have a Pollucom f heat meter and want to track the heat amount.
The display though is active for 1 second and turns off for 4 seconds.
Therefore I have no continuous display of my values and it is coincidence whether the picture is taken while display is on or off.
I tried to align by time (reboot until I have a display) but the intervall seems to be off.

Anyone a good idea, how to identify a display and make a picture in this exact moment?

**Comment by jomjol** [comment]
This device is not suitable for a time depended display. The alignment and the image recognition takes typically several 10s of seconds. Therefore a timing does not work.

**Comment by stefanovich1** [comment]
Well I get that.
To be clear. I do not want to make a reading every 4 seeconds.
I just need to cacht the value display within the 4 seconds. 
As you said timing is not the key. 
I was thinking about things like:
- 3 second expousure (can only set 1200ms)
- during one flow make 1 picture every second for 4 seconds.....batch analyse it afterwards

are those Ideas too off? 
Can I set the expusure time for experimental reasons to 4000ms?

**Comment by stefanovich1** [comment]
Hey I got a question.
I can define different flows.
This is used to read energy meters with changing tariffs.
Usually thos tariffs are shown in one display alternating. 

Therefore I assume someone had a good idea alread to work with blinking, flashing or interchangeable displays. 

But again: can someone give me a hint how to overwrite the 1200ms for the expouser time?
changing it in the config.ini has no effect....I think if the value is above 12000ms it drops back to standard value indipendent of the config.ini

**Comment by jomjol** [comment]
Can you tell me, where you set the exposure time in the setting? If you mean the AEC paramater, this is not a time, that you insert there. 
Please check the camera documentation to see, what it means (I don't know myself).
It is not planned to have this kind of exposure controll for the camera.
